src/app/admin/index.ts

```diff
@@ -0,0 +1,17 @@
+import { checkedToUsername } from "@domain/users"
```

Comment:
not sure about file location, if this is `getUserByUsername` use case then should be in user folder

Reply:
I've been thinking about it, but the only use case is from the admin panel?